(GH-1789) Rescue EMFILE error with helpful message (#1825)
puppetlabs/bolt: documentation/bolt_known_issues.md

## Limited Kerberos support

Support for Kerberos over WinRM from a Linux host is currently experimental and requires the [MIT Kerberos library](https://web.mit.edu/Kerberos/www/krb5-latest/doc/admin/install_clients.html) to be installed. In the future, Bolt will support Kerberos when running on Windows ([BOLT-1323](https://tickets.puppet.com/browse/BOLT-1323)) and macOS ([BOLT-1471](https://tickets.puppet.com/browse/BOLT-1471)).

## Errno::EMFILE Too many open files

> This error is raised when there are too many files open in Bolt's ruby process. You can see what
> your current limit is by running

Reviewer: ```suggestion
This error is raised when there are too many files open in Bolt's Ruby process. To see what
your current limit is, run:
```

```
ulimit -n
```

> and can raise the limit by setting the following in your shell configuration (ie.
> `~/.bash_profile`):

Reviewer: ```suggestion
To raise the limit, set the following in your shell configuration file (For example,
`~/.bash_profile`):
```
